includes/class-newspack-popups-model.php

```diff
@@ -231,6 +231,8 @@ public static function retrieve_preview_popup( $post_id ) {
 				'trigger_type'            => filter_input( INPUT_GET, 'trigger_type', FILTER_SANITIZE_STRING ),
 				'trigger_delay'           => filter_input( INPUT_GET, 'trigger_delay', FILTER_SANITIZE_STRING ),
 				'trigger_scroll_progress' => filter_input( INPUT_GET, 'trigger_scroll_progress', FILTER_SANITIZE_STRING ),
+				'trigger_posts_count'     => filter_input( INPUT_GET, 'trigger_posts_count', FILTER_SANITIZE_STRING ),
```

Comment:
The other `trigger_*` meta values denote what triggers the prompt display, not insertion logic. How about `archive_insertion_posts_count`?

Reply:
Yes, make sense 👍